File: src/main/java/com/github/sorusclient/client/adapter/event/ChatReceivedEvent.java
/*
 * Copyright (c) 2022. Danterus
 * Copyright (c) 2022. Sorus Contributors
 *
 * SPDX-License-Identifier: MPL-2.0
 */

package com.github.sorusclient.client.adapter.event;

import com.github.sorusclient.client.adapter.IText;
import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
public class ChatReceivedEvent {

    private @Getter final String message;
    private @Getter final IText text;

}

File: src/main/kotlin/com/github/sorusclient/client/adapter/event/ChatReceivedEvent.kt
/*
 * Copyright (c) 2022. Danterus
 * Copyright (c) 2022. Sorus Contributors
 *
 * SPDX-License-Identifier: MPL-2.0
 */

package com.github.sorusclient.client.adapter.event

import com.github.sorusclient.client.adapter.IText

class ChatReceivedEvent(val message: String, val text: IText)